packages/connectivity/src/scp-cf/destination/destination-accessor-provider-subscriber-lookup.spec.ts

```diff
@@ -296,3 +350,83 @@ describe('jwtType x selection strategy combinations. Possible values are {subscr
     });
   });
 });
+
+describe('call getAllDestinations with and without subscriber token', () => {
+  beforeEach(() => {
+    mockThingsForCombinations();
+  });
+
+  afterEach(() => {
+    nock.cleanAll();
+    jest.clearAllMocks();
+  });
+
+  const options: AllDestinationOptions = {
+    iasToXsuaaTokenExchange: false
+  };
+
+  it('should fetch all subscriber destinations', async () => {
+    const logger = createLogger({
+      package: 'connectivity',
+      messageContext: 'destination-accessor'
+    });
+    const debugSpy = jest.spyOn(logger, 'debug');
+
+    const allDestinations = await getAllDestinationsFromDestinationService({
+      ...options,
+      jwt: subscriberUserJwt
+    });
+
+    expect(allDestinations).toEqual([parsedSubscriberDestination]);
+
+    expect(debugSpy).toHaveBeenCalledWith(
+      'Retrieving all destinations for account: "subscriber" from destination service.'
+    );
+    expect(debugSpy).toHaveBeenCalledWith(
+      'Retrieving subaccount destination: DESTINATION.\n'
+    );
+  });
+
+  it('should fetch all provider destinations when called without anything', async () => {
```

Comment:
[pp] without anything is perhaps a misleading doupble negation `without passing a JWT` perhaps?